### ISSUE TITLE
Vertical Alignment

### ISSUE THREAD

**Issue by ygorlf** [Mwni/pixi-text-input#22]
Hi @Mwni, thanks for the effort on this library, really useful :)

Do you plan to support vertical alignment?

**Comment by Mwni** [comment]
you mean to specify a height for the textfield and have the text center accordingly?

**Comment by ygorlf** [comment]
yes, on my components I use this alignment a lot, It's a very useful feature
